### ISSUE TITLE
Duplicate ship_tank_karavan.png

### ISSUE THREAD

**Issue by kervala** [kaetemi/ryzomcore-assets#11]
ship_tank_karavan.png is present in 2 directories :

- stuff/caravan/agents/_textures/accessories
- stuff/caravan/agents/_textures/actors

There are slightly differences between them and I think the good one is in "accessories", because this texture has nothing to do with "actors" :p

We should keep only one of them and remove the other one.

**Comment by kaetemi** [comment]
The ship is an actor.

Accessories is for hand-held equipment (like swords) and non-moving dynamically placed objects.
- Actors: Movable entity (including armors) (anything with a skeleton basically)
- Accessories: Hand-held equipment (swords) and non-moving dynamically placed objects (anything without a skeleton)
- Decors: Static models (lightmapped)
- Construction: Decors XRef'd by other files (landscape), not directly exported

**Comment by kaetemi** [comment]
Ship models are under actor as well, but textures all seem misplaced. :)

Please do double check with live data which of the textures is the latest version.

**Comment by kervala** [comment]
Yep, that's why I didn't remove it directly and prefer to create an issue :) If files were identical, there wouldn't have any problem :p

**Comment by kaetemi** [comment]
The trame.png under caravan actors also needs to be deleted.

**Comment by kaetemi** [comment]
The caravan ships are a special case, they are under actors because they do have a skeleton for some animations, and can move (also with specific movement animations and effects.)
